waspc/src/Wasp/AppSpec.hs

```diff
@@ -53,7 +54,9 @@ data AppSpec = AppSpec
     -- If false, it means project is being compiled for development purposes (e.g. "wasp start").
     isBuild :: Bool,
     -- | The contents of the optional user Dockerfile found in the root of the wasp project source.
-    userDockerfileContents :: Maybe Text
+    userDockerfileContents :: Maybe Text,
+    -- | A list of paths to any config files found (e.g., tailwind.config.js) and where to copy them.
+    configFiles :: [ConfigFileRelocator]
```

Comment:
Instead of adding an explicit mapping for just 2 config files of interest, I decided to plan ahead for other configs we may want to copy over to web app/server.

Reply:
I may lean toward as-is, since they are added by users, but maybe don't really "belong" to users, if that makes sense?

Reply:
Makes sense!